src/DynamoCore/Library/FunctionDescriptor.cs

```diff
@@ -222,6 +224,33 @@ public string Category
             {
                 var categoryBuf = new StringBuilder();
                 categoryBuf.Append(GetRootCategory());
+                
+                //if this is not BuiltIn function search NodeCategoryAttribute for it
+                if(ClassName!=null)
+                {
+                    //get class type of function
+                    var type = AppDomain.CurrentDomain.GetAssemblies().First(x => x.GetName().Name == Path.GetFileNameWithoutExtension(Assembly)).GetType(ClassName);
```

Comment:
Please can we break the long line, as you do below, it makes it quite a bit easier to read